frontend/src/components/LanguageSelectorItem.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+class LanguageSelector extends PureComponent {
```

Comment:
LanguageSelectorItem is better, isn't it?

Reply:
That's strange, I was using LanguageSelectorItem. Really don't know what happened.